Share The Load/Screens/ViewController.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  ViewController.swift
+//  Share The Load
+//
+//  Created by Christopher Hefferman on 10/01/2021.
+//
+
+import UIKit
+import Firebase
+import RealmSwift
+
+class ViewController: UIViewController {
+    
+    private let getRealmQuestionsButton = UIButton()
+    private let removeFromRealmButton = UIButton()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        fetchQuiz()
+        createGetRealmQuestionsButton()
+        createRemoveButton()
+    }
+    
+    
+    private func createGetRealmQuestionsButton() {
+        view.addSubview(getRealmQuestionsButton)
+        getRealmQuestionsButton.translatesAutoresizingMaskIntoConstraints = false
+        getRealmQuestionsButton.setTitle("Get Questions From Realm", for: .normal)
+        getRealmQuestionsButton.frame = view.frame
+        getRealmQuestionsButton.backgroundColor = .systemBlue
+        getRealmQuestionsButton.layer.cornerRadius = 20
+        getRealmQuestionsButton.addTarget(self, action: #selector(buttonTapped), for: .touchUpInside)
+        
+        NSLayoutConstraint.activate([
+            getRealmQuestionsButton.centerYAnchor.constraint(equalTo: view.centerYAnchor),
+            getRealmQuestionsButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            getRealmQuestionsButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+            getRealmQuestionsButton.heightAnchor.constraint(equalToConstant: 50)
+        ])
+    }
+    
+    
+    private func createRemoveButton() {
+        view.addSubview(removeFromRealmButton)
+        removeFromRealmButton.translatesAutoresizingMaskIntoConstraints = false
+        removeFromRealmButton.setTitle("Remove Question", for: .normal)
+        removeFromRealmButton.frame = view.frame
+        removeFromRealmButton.backgroundColor = .systemRed
+        removeFromRealmButton.layer.cornerRadius = 20
+        removeFromRealmButton.addTarget(self, action: #selector(removeButtonTapped), for: .touchUpInside)
+        
+        NSLayoutConstraint.activate([
+            removeFromRealmButton.topAnchor.constraint(equalTo: getRealmQuestionsButton.bottomAnchor, constant: 20),
+            removeFromRealmButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            removeFromRealmButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+            removeFromRealmButton.heightAnchor.constraint(equalToConstant: 50)
+        ])
+    }
+    
+    
+    @objc func buttonTapped() {
+        let persistanceService = PersistanceService()
+        let questionsFromRealm = persistanceService.getQuestions()
+        
+        for question in questionsFromRealm {
+            print("Question: \(question.title) || Answer: \(question.correctAnswer)")
+        }
+    }
+    
+    
+    @objc func removeButtonTapped() {
```

Comment:
It's good to get in the habit of making all these private

Reply:
Thanks. I've made these private now. So would you recommend that any methods I don't plan to use outside the scope of a file to be private?

Reply:
Definitely, the only things which aren't private should be properties or functions which you explicitly want other objects to be able to view or call.

All outlets should be private too, if you have a text label outlet, and you want the text to be set by another object, then you'd create a public string in your class which sets the label's text for you.